api-controllers/ServiceBrokerApiController.js

```diff
@@ -215,7 +216,8 @@ class ServiceBrokerApiController extends FabrikBaseController {
           return eventmesh.apiServerClient.getOSBResourceOperationStatus(lastOperationState);
         }
       })
-      .then(done);
+      .then(() => _.get(context, 'plan.manager.settings.dashboard_url_template') !== undefined ? eventmesh.apiServerClient.waitTillInstanceIsScheduled(req.params.instance_id) : {})
```

Comment:
update does not need it. You'll have `clusterId` set already.

Reply:
Done it considering in future if operators want to change dashboard on plan update etc.